ragna/deploy/_ui/modal_configuration.py

```diff
@@ -272,7 +290,7 @@ def advanced_config_ui(self):
                         "border-left": "1px solid var(--neutral-stroke-divider-rest)"
                     },
                 ),
-                height=250,
+                width=800,
```

Comment:
Height is dynamic (see earlier comments on the new `param` object introduced) so no longer needs to be fixed. Width is pinned to modal width just to be on the safe side.